### PR TITLE
Add failing rbx test for EndpointResolver

### PR DESCRIPTION
**Pull request by tak1n**
Currently Rbx has problems with classes which extend `SimpleDelegator`.

See rbx issue https://github.com/rubinius/rubinius/issues/3451 for further details.

To get a better grasp what exactly was failing I added this test.

### CODE REVIEW

**Comment by tak1n** [comment]
Seems like on travis everything is passing, closing therefore.
Have to investigate why on my local rbx install the tests are failing.

**Comment by tak1n** [comment]
Hm weird..

```
router ➤ ruby -v                                                                                                                                                                                        git:master
rubinius 2.5.7.c21 (2.1.0 a8dcb34b 2015-06-25 3.5.0 JI) [x86_64-linux-gnu]
router ➤ ruby -Ilib:test test/routing/endpoint_resolver_test.rb                                                                                                                                         git:master
Run options: --seed 50923

# Running:

..............F

Finished in 0.011274s, 1330.5018 runs/s, 1330.5018 assertions/s.

  1) Failure:
Lotus::Routing::EndpointResolver::endpoint with nested routes#test_0001_reponds to :routes [/home/benny/Development/ruby/lotus/router/test/routing/endpoint_resolver_test.rb:96]:
Expected: true
  Actual: false

15 runs, 15 assertions, 1 failures, 0 errors, 0 skips
```

Travis rbx version is `rubinius 2.5.7 (2.1.0 d4fdeaad 2015-06-17 3.4 JI) [x86_64-linux-gnu]` therefore a little bit older then mine.
So when travis / rvm updates to a newer rbx this will fail, reopening.

File: test/routing/endpoint_resolver_test.rb [hunk 1]
```diff
@@ -78,6 +78,29 @@
...
+
+        def routes
+          Lotus::Router.new do
+            get '/home', to: 'home#index'
+          end
+        end
+      end
+    end
+
+    it 'reponds to :routes' do
```
Comment (joneslee85) [inline test/routing/endpoint_resolver_test.rb]: s/reponds/responds

**Comment by joneslee85** [comment]
looks good to me :+1:

File: test/routing/endpoint_resolver_test.rb (hunk 1 above)
Comment (tak1n) [inline test/routing/endpoint_resolver_test.rb]: oops :D

**Pull request merged**